### ISSUE TITLE
Long CLI options to 6to5-node doesn't work

### ISSUE THREAD

**Issue by vtambourine** [6to5/6to5#679]
Long options to `6to5-node` considered as filename that doesn't exists.

For example, calling `6to5-node` in experimental mode rises an error:
```
$ 6to5-node --experimental server.js
Error: Cannot find module '/Users/benjamin/Projects/hypervisor/--experimental'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/6to5/bin/_6to5-node:92:12)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

However, short name for that option works well:
```
$ 6to5-node -r server.js
```

Application version is 3.3.4

**Comment by vtambourine** [comment]
Probably, this is the same issue as #667?

**Comment by vtambourine** [comment]
Thank you!

**Comment by sebmck** [comment]
Fixed as of 3.3.7, thanks!
